Play44, Video44 and Streamcloud

smokdpi opened the pull request:
Play44

    added "byzoo.org", "playpanda.net", "playbb.me", "videozoo.me" support
    removed unnecessary logging, and unused code
    Class changed to Play44Resolver was FilenukeResolver

Video44

    added "easyvideo.me" support
    removed unnecessary logging, and unused code
    Class changed to Video44Resolver was FilenukeResolver

StreamCloud

    added additional url support
    removed logo, popups, unused imports and code

Eldorados commented:
Play44 & Video44 look good, but I think streamcloud should be dropped from your pull

smokdpi commented:
I've fixed the the readability of the single line statements. 
From the the current the changes include:

unused imports in current streamcloud:
- from lib import jsunpack
- import os 
- import xbmcgui
- import time

unused code:
- dialog = xbmcgui.Dialog()

- multiple regex references, changed over to self.pattern

URL supported:

old: http://www.streamcloud.eu/aSDdadf34
	or http://streamcloud.eu/aSDdadf34 
new: the old as well as http://www.streamcloud.eu/aSDdadf34/
	or http://www.streamcloud.eu/aSDdadf34/this.is.the.video.html
	(with or without www of course)

If you would still like it dropped from the pull I will resubmit the others. A couple sites I use myself use the new url types added.
